packages/autorest.typescript/test/unit/generators/static/readmeFileGenerator.spec.ts

```diff
@@ -79,64 +79,10 @@ describe("readmeFileGenerator", () => {
         /(\r\n|\n|\r)/gm,
         " "
       );
-      writeFileSync(path.join(__dirname, "files/case-hlcReadme.md"), getFirstFileContent(project));
-      assert.strictEqual(actualContents, expectedContends);
-    });
-
-    it("should generate rest-level client readme file correctly", async () => {
```

Comment:
why remove this test ?

Reply:
This is the unit test for readmeFileGenerator.ts file and considering we move the logic to rlc-codegen folder so this UT will be useless.

Reply:
Added it back